node/src/main/scala/coop/rchain/node/api/WebApi.scala

```diff
@@ -335,23 +331,55 @@ object WebApi {
   private def exprFromBundleProto(b: Bundle): Option[RhoExpr] = exprFromParProto(b.body)
 
   private def mkUnforgExpr(f: String => RhoUnforg, bs: ByteString): ExprUnforg =
-    ExprUnforg(f(toHex(bs)))
+    ExprUnforg(f(bs.toHexString))
 
   // RhoExpr to protobuf
 
+  private def rhoExprToParProto(exp: RhoExpr): Par = exp match {
+    // Nested expressions (Par, Tuple, List and Set are converted to JSON list)
+    case data: ExprPar   => exprParToParProto(data)
```

Comment:
Is there a reason to use different _case_ style for this case?

Reply:
With `case ExprPar(data)` data has type `List[RhoExpr]`. I little bit [confused](https://github.com/rchain/rchain/pull/3718#discussion_r892721910) :thinking: As I understood we should not process `ExprPar` as list of Expr.